modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+
+=== Install the Connector from the Studio Taskbar
+
+. In Studio, create a Mule project.
+. Click the Exchange icon *(X)* in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for the connector, for example, *Amazon SQS*.
+. Select the connector.
+. Click *Add to project*.
+. Follow the prompts to install the connector.
+
+=== Install the Connector from the Mule Palette View
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, enter the connector name in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source for the Connector
+
+An input source starts a flow when a specified condition is met. Configure an input source for the connector, such as an HTTP Listener operation (HTTP Listener), Scheduler endpoint (Scheduler), or connector-specific input source.
+
+=== Use HTTP Listener as an Input Source
+
+HTTP Listener listens for HTTP requests and triggers the associated flow when it receives them. To use HTTP Listener as an input source:
+
+. In the Mule Palette view, search for *HTTP* and select the *Listener* operation:
++
+image::introduction/intro-config-studio-select-listener.png[Select HTTP Listener as an Input Source]
++
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to add a global element for HTTP Listener.
++
+The following screenshot shows the default values for the HTTP Listener global element:
++
+image::introduction/intro-config-studio-listener.png[Global Element Screen for HTTP Listener]
++
+. Accept the defaults on the *General* tab or specify required information:
+* To listen on a specific port for every network interface on the server, leave the default value `0.0.0.0` for the *Host* property.
+* To listen for incoming requests generated inside the server, set the *Port* value to `localhost`.
+. To use TLS, access the *TLS* tab and specify the required information.
+. On the *Advanced* tab, specify a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified server.
+. Click *OK*.
+. Set the *Path* field to the appropriate path.
+. Select the other tabs and specify the appropriate field values.
+
+For more information about HTTP Listener, see the xref:http/http-documentation.adoc[HTTP Connector Reference].
+
+=== Use Scheduler as an Input Source
+
+Use Scheduler to trigger a flow when a time-based condition is met. You can configure Scheduler to trigger a flow at a regular interval or you can use a more flexible cron expression. For example, you can use Scheduler to trigger a flow every five seconds, or you can set up a cron expression to trigger a flow at noon everyday for one month.
```

Comment:
Should this be "the Scheduler source" or something? Without an article, it's like a product name, but it's not a product name.